File: Bus Conductor.java
class Solution {
    public static int findMoves(int n, int[] ch, int[] pa) {
        Arrays.sort(ch);
        Arrays.sort(pa);
        int ans = 0;
        for(int i=0;i<ch.length;i++){
            while(ch[i]!=pa[i]){
                if(ch[i]<pa[i]) pa[i]--;
                else pa[i]++;
                ans++;
            }
        }
        return ans;
    }
}

File: Bus Conductor.cpp
class Solution {
  public:
    int findMoves(int n, vector<int> c, vector<int> p) {
        vector<int>v ; 
        sort(c.begin(),c.end()) ; // 2 2 6 6 
        sort(p.begin(),p.end()) ; // 1 2 3 6
        int sum=0;
        
        for(int i=0;i<n;i++){
            sum=sum+abs(c[i]-p[i]) ; 
        }
        return sum ;
    }
};
